sematic/__init__.py

```diff
@@ -21,6 +21,9 @@
 import sematic.types  # noqa: F401,E402
 from sematic.calculator import func  # noqa: F401,E402
 from sematic.container_images import has_container_image  # noqa: F401,E402
+from sematic.external_resource_plugins.timed_message import (  # noqa: F401,E402
+    TimedMessage,
+)
```

Comment:
There is a `plugins` directory. Maybe move that there instead?

The current pattern is
```
sematic/plugins/
    abstract_<scope>.py
    <scope>/
        <plugin_name>.py
```
for example:
```
sematic/plugins/
    abstract_storage.py
    storage/
        local_storage.py
        s3_storage.py
```
So this could be
```
sematic/plugins/
    abstract_external_resource.py
    external_resource/
        time_message.py
```

Where `abstract_external_resource.py` is essentially `sematic/external_resource.py`.

Reply:
Makes sense! That directory didn't exist when this code was first written.